tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Immutable from 'immutable';
+import * as helloWorldActionCreators from '../actions/helloWorldActionCreators';
+
+function select(state) {
+  // Which part of the Redux global state does our component want to receive as props?
+  return { $$helloWorldStore: state.$$helloWorldStore };
+}
+
+class HelloWorld extends React.Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    $$helloWorldStore: PropTypes.instanceOf(Immutable.Map).isRequired,
+  }
+
+  render() {
+    const { dispatch, $$helloWorldStore } = this.props;
+    const actions = bindActionCreators(helloWorldActionCreators, dispatch);
+    return (
+      <HelloWorldWidget $$helloWorldStore={$$helloWorldStore} actions={actions} />
```

Comment:
@justin808 I can do that, although I'm also fine with just changing the actual code to use the short syntax